.bloodhound.yml

```diff
@@ -1,9 +1,9 @@
 # Kubernetes versions
 # (e.g. 1.23.0 - has to match versions in https://github.com/yannh/kubernetes-json-schema)
 k8s_versions:
-  - 1.30.5
   - 1.31.1
   - 1.32.0
+  - 1.32.2
```

Comment:
add a 1.33.x version here

Reply:
looks like i got ahead of myself - there is no 1.33.x yet!

lets remove duplicate 1.32.x versions and retain the 1.30.x here.